jenkins/run-pytests

```diff
@@ -22,9 +22,7 @@ if [[ ":${PYTHONPATH:-}:" != *":${_pdir}:"* ]]; then
     export PYTHONPATH=${_pdir}${PYTHONPATH:+:${PYTHONPATH}}
 fi
 
-printf -- "black --check ...\n"
-black --check .
-printf -- "\nflake8 --verbose ...\n"
-flake8 --verbose .
-printf -- "\n"
-pytest ./lib/pbench/test/unit/common ./lib/pbench/test/unit/agent ./lib/pbench/test/unit/server ./lib/pbench/test/functional
+printf -- "lint check ... \n"
+tox -e lint
```

Comment:
This is already run under tox, see `tox.ini` for the `jenkins-pytests` environment.  See PR #1936.